examples/plugins/Android.bp

```diff
@@ -0,0 +1,15 @@
+cc_library {
+    name: "libsample_pcm_plugin",
+    vendor: true,
+    srcs: ["sample_pcm_plugin.c"],
+    cflags: ["-Werror", "-Wno-unused-parameter"],
+    header_libs: ["tinyalsa_headers"],
+}
+
+cc_library {
+    name: "libsample_mixer_plugin",
```

Comment:
Please rename to libtinyalsav2_example_plugin_mixer

Reply:
done